api/specs/web-server/requirements.txt

```diff
@@ -1,5 +1,7 @@
 # Extra reqs, besides webserver's
 
-fastapi==0.96.0
+fastapi
```

Comment:
Q: should we not pin at least `fastapi` and `pydantic`? At least have it in sync with the versions from `models-library`? This would allow for a more reliable generation of the specs.